examples/flutter_gallery/lib/demo/cupertino/cupertino_text_field_demo.dart

```diff
@@ -111,6 +112,8 @@ class _CupertinoTextFieldDemoState extends State<CupertinoTextFieldDemo> {
         border: Border(bottom: BorderSide(width: 0.0, color: CupertinoColors.inactiveGray)),
       ),
       placeholder: 'Location',
+      uniqueIdentifier: 'location',
```

Comment:
As we discussed in the emails, we can aim to remove them. If it is only used in Android platform, can we try to use a way to carry this to Adnroid platform. For example creating a stable ID per field in the platform ?

Reply:
If the flutter input fields only have unique but unstable identifiers, it could be difficult to assign stable and unique ids to their platform counterparts: when I generate a new ID for a flutter input field, how would the platform side know that this _new_id_ should be tied to the native input field that has _old_id_?

Reply:
The stable requirement comes from https://github.com/flutter/flutter/issues/49385. I think we can make the unique identifier optional, but then it would be too easy to overlook.